### PR TITLE
 K8s - Getting the source IP behind the LoadBalancer  

### PR DESCRIPTION
**Pull request by LostInBrittany**
- New tutorial: Getting the source IP behind the LoadBalancer
- Fixing typos...

### CODE REVIEW

File: pages/platform/kubernetes-k8s/available-upcoming-features/guide.en-gb.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ We list here the most frequently requested OVH Managed Kubernetes features that
 - **Load Balancer** Use the free preview of our integrated load balancer to expose your services on any port. Details available in the [exposing your services](../using-lb/) guide.
 - **Pod autoscaling** Use the  [standard horizontal pod autoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/){.external} to distribute dynamically your containerized workloads on a given set of worker nodes.
 - **RBAC** Use the standard [RBAC authorization](https://kubernetes.io/docs/reference/access-authn-authz/rbac/){.external} mode to distribute rights within your organization and/or assure compatibility with specific applications.
+- **New versions** : We will support upcoming versions, during the quarter following their official release. We also propose managed version upgrades.
```
Comment (thcdrt) [inline pages/platform/kubernetes-k8s/available-upcoming-features/guide.en-gb.md]: Maybe we should use present here.
s/We will support/We support

File: pages/platform/kubernetes-k8s/backing-up-a-cluster/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+This tutorial presupposes that you already have a working OVH Managed Kubernetes cluster, and some basic knowledge of how to operate it. If you want to know more on those topics, please look at the [deploying a Hello World application](../deploying-hello-world/) documentation.
+
+You also need to have [Helm](https://docs.helm.sh/){.external} installed on your workstation and your cluster, please refer to the [How to install Helm on OVH Managed Kubernetes Service](../installing-helm/) tutorial.
+
+## Velero
+
+As explained in [Velero](https://velero.io){.external} website, *Velero is an open source tool to safely backup and restore, perform disaster recovery, and migrate Kubernetes cluster resources and persistent volumes*.
+
+By default Velero supports a variety of storage providers, and it has a plugin system for easily  additional backup and volume storage platforms. In this tutorial we are using this extensibility capacities by installing the [Plugin Velero Openstack](https://github.com/cisco-sso/velero-plugin-Openstack){.external} allowing Velero to use Openstack as both backup and volume storage platform.
```
Comment (thcdrt) [inline pages/platform/kubernetes-k8s/backing-up-a-cluster/guide.en-gb.md]: There is too much space between "easily" and "additional"

File: pages/platform/kubernetes-k8s/backing-up-a-cluster/guide.en-gb.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Installing Velero on OVH Managed Kubernetes
```
Comment (thcdrt) [inline pages/platform/kubernetes-k8s/backing-up-a-cluster/guide.en-gb.md]: Is that normal that this section is empty ?

File: pages/platform/kubernetes-k8s/getting-source-ip-behind-loadbalancer/guide.en.gb.md [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+The easiest way to deploy services behind the Load Balancer while keeping the source IP is to place your services under an `Ingress`, itself behind the `LoadBalancer`. 
+
+The `Ingress` is exposed to the outside of the cluster either via  `LoadBalancer`, and it routes incoming traffic to your services according to configured rules. And additional advantage of this setting is the cost: you can have lots of services behind a single `LoadBalancer`.
+
+In this tutorial we are using the most basic Ingress Controller: [NGINX Ingress Controller](https://github.com/kubernetes/ingress-nginx){.external}, where an NGINX server take the role of reverse proxy.
+
+
+### 1. Installing the NGINX Ingress Controller
+
+The official way to install the **NGINX Ingress Controller* is using a mandatory manifest file:
```
Comment (thcdrt) [inline pages/platform/kubernetes-k8s/getting-source-ip-behind-loadbalancer/guide.en.gb.md]: I think it miss a star at the end of 'NGINX Ingress Controller'

**Review by thcdrt — COMMENTED** [review]
Thanks for this wonderful doc Horacio !
I just found some little typos.

File: pages/platform/kubernetes-k8s/available-upcoming-features/guide.en-gb.md (hunk 1 above)
Comment (LostInBrittany) [reply pages/platform/kubernetes-k8s/available-upcoming-features/guide.en-gb.md]: Fixing, thanks!

File: pages/platform/kubernetes-k8s/backing-up-a-cluster/guide.en-gb.md (hunk 2 above)
Comment (LostInBrittany) [reply pages/platform/kubernetes-k8s/backing-up-a-cluster/guide.en-gb.md]: Fixing, thanks!

File: pages/platform/kubernetes-k8s/backing-up-a-cluster/guide.en-gb.md (hunk 3 above)
Comment (LostInBrittany) [reply pages/platform/kubernetes-k8s/backing-up-a-cluster/guide.en-gb.md]: Oh, this guide isn't ready yet, I think I committed it by error, deleting it :)

File: pages/platform/kubernetes-k8s/getting-source-ip-behind-loadbalancer/guide.en.gb.md (hunk 4 above)
Comment (LostInBrittany) [reply pages/platform/kubernetes-k8s/getting-source-ip-behind-loadbalancer/guide.en.gb.md]: Fixing, thanks!

**Review by LostInBrittany — COMMENTED** [review]
A new commit it coming, fixing ewrrors found by @thcdrt

File: pages/platform/kubernetes-k8s/getting-source-ip-behind-loadbalancer/guide.en.gb.md [hunk 5]
```diff
@@ -0,0 +1,309 @@
...
+Now you can test it using the LoadBalancer URL:
+
+```bash
+curl  xxxxxxxxxx.lb.c1.gra.k8s.ovh.net
+```
+
+And you should get the HTTP parameters of your request, including the right source IP in the `x-real-ip` header:
+
+
+{
```
Comment (thcdrt) [inline pages/platform/kubernetes-k8s/getting-source-ip-behind-loadbalancer/guide.en.gb.md]: This json could be better formatted.

**Pull request merged**